addons/mail/static/src/models/rtc_call_participant_card.js

```diff
@@ -67,6 +69,21 @@ registerModel({
                 return `/mail/channel/${this.channel.id}/guest/${this.invitedGuest.id}/avatar_128?unique=${this.invitedGuest.name}`;
             }
         },
+        /**
+         * @private
+         * @returns {string}
+         */
+        _computeInboundConnectionTypeText() {
+            if (!this.rtcSession || !this.rtcSession.localCandidateType) {
+                return this.env._t("no connection");
+            }
+            return sprintf(
+                this.env._t('Inbound: %(candidateType)s (%(protocol)s)'), {
+                    candidateType: this.rtcSession.remoteCandidateType,
+                    protocol: this.rtcSession.remoteCandidateType === 'relay' ? "TURN": "STUN",
```

Comment:
strange to have 2 different values that are a representation of the same data?

Reply:
there are mutiple `candidateType` that use STUN, so one indicates the precise type which is quite technical and the other indicates the traversal protocol